ci/code_checks.sh

```diff
@@ -152,6 +152,10 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -R --include="*.py" --include="*.pyx" -E "class\s\S*\(object\):" pandas scripts
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    MSG='Check for empty class parenteses' ; echo $MSG
+    invgrep -R --include="*.py" --include="*.pyx" -E "class\s\S*\(\):" pandas scripts
```

Comment:
I think you can consolidate this with the previous check for inheriting from `object` with a regex pattern along the lines of `class\s\S*\((object)?\):`